docs/claiming.rst

```diff
@@ -65,6 +67,8 @@ Claim Deposit
 In order to claim a request you must put down a deposit.  This deposit amount
 is equal to twice the ``payment`` amount associated with this request.
```

Comment:
This will need to be updated with the introduction of `claimData.requiredDeposit`

Reply:
On it